### DISCUSSION TITLE
Slaesh's CC2652RB stick doesn't support zigbee 3.0?

### DISCUSSION THREAD

**Discussion by Sjworks** [Koenkk/zigbee2mqtt#14802] [Devices]
I bought three Gledopto controllers.
These devices are zigbee 3.0(pro).

- GL-C-006P : https://www.zigbee2mqtt.io/devices/GL-C-006P.html
- GL-C-009P : https://www.zigbee2mqtt.io/devices/GL-C-009P.html
- GL-C-001P : https://www.zigbee2mqtt.io/devices/GL-C-008P.html#gl-c-001p

And I running Zigbee2Mqtt with Slaesh's CC2652RB stick.
Above three devices is my very first zigbee 3.0 devices.

The problem is, I power up these devices and tried to pairing, but nothing happens.
Also, I couldn't see any logs.(Of course, I checked the debug level log)

I thought it was a problem with the device, so I tried pairing it with Hue Bridge, and it was connected normally.
Devices are looks normal.
What is suspicious is that the Dongle I am using does not support zigbee3.0.

Has anyone experienced the above problem?

![1667534892915](https://user-images.githubusercontent.com/773003/199883584-9c479286-9de9-464f-8f48-1fa1727f7a2b.jpg)

**Comment by lux73** [comment]
Try first updating your Coordinator - yours is on a very old Firmware: 20210120

Mine ist actually on 20220219

There are many improvements with the CC2652RB between these Updates

**Reply by Sjworks** [reply]
@lux73 
Thanks for your help.
After updating to the latest version (`20220219`), I tried again and it works fine.

The Coordinator type is marked as `zStack3x0`, so I thought it was not a firmware version problem, but there seems to be a problem with the old version.

**Reply by lux73** [reply]
nice 👍
